### PR TITLE
US72282 - move common libraries to al-collector-js

### PR DESCRIPTION
**Pull request by ikemsley**
Move common libraries from this repo to [al-collector-js](https://github.com/alertlogic/al-collector-js).

### CODE REVIEW

**Comment by ikemsley** [comment]
@tomdos @kkuzmin @alexturkin Please CR.

**Comment by tomdos** [comment]
@ikemsley Don't forget to move libs to `al-collector-js` ;-)

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "cfn-response": "*",
     "moment": "^2.19.2",
     "request": "*",
-    "request-promise-native": "*"
+    "request-promise-native": "*",
+    "al-collector-js": "git://github.com/alertlogic/al-collector-js#master"
```
Comment (kkuzmin) [inline package.json]: Let's call the dependency as just `al-collector`.

Comment (kkuzmin) [reply package.json]: What about publishing the package in NPM https://www.npmjs.com/

Comment (tomdos) [reply package.json]: I think it is fine as we agreed not to do it unless it is really necessary for npm usage.

**Comment by ikemsley** [comment]
@kkuzmin I changed the package name from `"al-collector-js"` to `"al-collector"` but now npm doesn't seem to be able to pull it down and build it.  I think I'll have to revert the last commit.  If you want it to be called "al-collector" then I think the library should be called that too.  Otherwise the whole thing falls apart quickly.  I propose to just leave it as `"al-collector-js"`.

**Comment by tomdos** [comment]
@ikemsley keep the name of both repo and library `al-collector-js` please. I think it is a bad idea to have one name for repo and another for library.

File: package.json [hunk 2]
```diff
@@ -26,7 +26,8 @@
     "cfn-response": "*",
     "moment": "^2.19.2",
     "request": "*",
-    "request-promise-native": "*"
+    "request-promise-native": "*",
+    "al-collector": "git://github.com/alertlogic/al-collector-js#master"
```
Comment (tomdos) [inline package.json]: We need dependency to al-aws-collector-js

**Pull request merged**